kolibri/core/discovery/management/commands/enumeratedrives.py

```diff
@@ -1,11 +1,7 @@
-import logging as logger
```

Comment:
@jamalex this whole command seems unused? Not called from code base, not in docs. Was it perhaps thought of as a debugging command?